main.py

```diff
@@ -508,28 +514,31 @@ def add_to_dict_of_sets(self, intf, circuit_id):
     @listen_to('kytos/topology.port.created')
     def load_evcs(self, event):
         """Try to load the unloaded EVCs from storehouse."""
-        log.debug("Event load_evcs %s", event)
-        circuits = self.storehouse.get_data()
-        if not self._circuits_by_interface:
-            self.load_circuits_by_interface(circuits)
-
-        interface_id = '{}:{}'.format(event.content['switch'],
-                                      event.content['port'])
-
-        for circuit_id in self._circuits_by_interface.get(interface_id, []):
-            if circuit_id in circuits and circuit_id not in self.circuits:
-                try:
-                    evc = self._evc_from_dict(circuits[circuit_id])
-                except ValueError as exception:
-                    log.info(
-                        f'Could not load EVC {circuit_id} because {exception}')
-                    continue
-
-                evc.deactivate()
-                evc.current_path = Path([])
-                evc.sync()
-                self.circuits.setdefault(circuit_id, evc)
-                self.sched.add(evc)
+        with self._lock:
+            log.debug("Event load_evcs %s", event)
+            circuits = self.storehouse.get_data()
+            if not self._circuits_by_interface:
+                self.load_circuits_by_interface(circuits)
+
+            interface_id = '{}:{}'.format(event.content['switch'],
+                                          event.content['port'])
+
+            for circuit_id in self._circuits_by_interface.get(interface_id,
+                                                              []):
+                if circuit_id in circuits and circuit_id not in self.circuits:
+                    try:
+                        evc = self._evc_from_dict(circuits[circuit_id])
+                    except ValueError as exception:
+                        log.info(
```

Comment:
```suggestion
                        log.error(
```
I think it should be logged as an error, because of the fact that an EVC that was supposed be load was actually not loaded.